### DISCUSSION TITLE
Supabase Realtime doesnt send any callback

### DISCUSSION THREAD

**Discussion by lucasgbsampaio** [supabase/supabase#5153] [Questions] [answered]
Hi. 

I made the configuration so that supabase listens to my database and sends all changes in real time. In local mode everything works perfectly, but in production no data inserted/updated goes through the supabase callback. All the configuration is right as in the readme.
I'm using realtime server (not RLS), should i migrate to RLS ?

Supabase Version: Latest
Client: Node.js
Postgres Version: 12

**Comment by w3b6x9** [comment, marked as answer]
@lucasgbsampaio are you self-hosting? try RLS and see if you still have the same issues.

**Reply by w3b6x9** [reply]
@lucasgbsampaio hey, just wanted to check in and see if it was working now.

**Reply by lucasgbsampaio** [reply]
@w3b6x9 yeah, its working now, thank you
